routes/api/roles/doc.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @swagger
+ * /users/:id:
+ *   put:
+ *     tags:
+ *       - Roles
```

Comment:
I think the `tags` here should be a `User` because this PR is concerning to the User